patch/config-delta-ag9064.patch

```diff
@@ -0,0 +1,44 @@
+diff --git a/debian/build/build_amd64_none_amd64/.config b/debian/build/build_amd64_none_amd64/.config
+index e9b840e..dea8b32 100644
+--- a/debian/build/build_amd64_none_amd64/.config
++++ b/debian/build/build_amd64_none_amd64/.config
+@@ -3090,13 +3090,13 @@ CONFIG_HVC_IRQ=y
+ CONFIG_HVC_XEN=y
+ CONFIG_HVC_XEN_FRONTEND=y
+ CONFIG_VIRTIO_CONSOLE=m
+-CONFIG_IPMI_HANDLER=m
++CONFIG_IPMI_HANDLER=y
```

Comment:
why do we need to change to built-in option? is kernel module ok?

Reply:
Hi, sir:
    Because to load the LPC and ipmi  module has some timing and sequence issue and they should be put into kernel without loadable. 

Hans